### ISSUE TITLE
IntAct IPI annotation to P03772

### ISSUE THREAD

**Issue by jimhu-tamu** [geneontology/go-annotation#1339]
http://www.ebi.ac.uk/QuickGO/GProtein?ac=P03772

The imported IntAct annotation is showing interaction between a bacteriphage protein (the annotation object) and a mouse protein (the with/from). This is not biologically relevant to the function of the phage protein and seems like something that might be catchable by taxon constraints.

**Comment by bmeldal** [comment]
The interaction is bona fide: bacteriophage lambda phosphatase dephosphorylates mouse Sak in an in vitro expt to test for Sak autophosphorylation activity. Whether you want this type of interaction in GO is your decision.

**Comment by pgaudet** [comment]
Hi @bmeldal 
I dont think this is correct according to the GO guidelines. 
You can say that bacteriophage lambda phosphatase has a phosphatase activity based on this assay;however what you are capturing here is the assay, not the physiologically relevant substrate. This interaction should not be in GO. 

Thanks, Pascale

**Comment by bmeldal** [comment]
Pascal, we curate whatever people do in the lab, whether it makes physiological sense of not. Do you know if you filter our GAF or do we have to change what we export? I've never been involved with the **IntAct** GAF, that was put in place long before I joined and hasn't changed for a long time. 

@sandraorchard will know more

**Comment by sandraorchard** [comment]
There is a lot of mixed species data generated in the PPI field and it would be very difficult to design filters which differentiated 'real' host-pathogen responses and not lab-induced. Bacteriophage-non-bacterial host I agree we could filter out, and we can look at ways of doing this in the export pipeline.

**Comment by pgaudet** [comment]
Right - you could probably get a combination of phage/viral species + host species and sort most of them out this way, is this what you mean ? 

Thanks, Pascale

**Comment by bmeldal** [comment]
I don't think it's an easy fix. It's perfectly possibly that a bona fide host-pathogen interaction has been shown in vitro - then there's not much we can deduce from the experimental conditions whether it's physiologically 'real' or not. I wonder if we need to add the "no uniprot export" tag to such interactions? 
@pporrasebi Pablo, what do you think?

**Comment by pporrasebi** [comment]
I agree with Sandra. In order to filter out what GO regards as non-physiological (weird experimental things such as human-yeast or mouse-bacteriophage interactions) one can use a pre-defined list of host-pathogen taxID pairs and create the appropriate filters for those. If GO can provide a list of the type of inter-species interactions they would find acceptable, we can certainly figure out a way to put the filters in the export pipeline.

**Comment by bmeldal** [comment]
@pporrasebi please advise on action - if you raise a Jira ticket for IntAct and link it to here I can close this ticket.

**Comment by pporrasebi** [comment]
Actiosn to be taken: 
1.- GO to produce a list of taxID pairs they would consider acceptable
2.- IntAct to implement the list as a filter in the GO IPI export

Anything else?

**Comment by bmeldal** [comment]
@pgaudet who would have to produce the list for us to tighten our export?

**Comment by pgaudet** [comment]
Hi @bmeldal 
The clean solution would be to tparse the host information in the UniProt entry - for example for 
http://www.uniprot.org/uniprot/P27958#names_and_taxonomy
they have this: 
![image](https://user-images.githubusercontent.com/4782928/33997911-419d4cba-e0e6-11e7-8723-f43aefa94318.png)

Can this be done ? 

Thanks, Pascale

**Comment by sandraorchard** [comment]
Hi Pascale
We have the taxonomy info, and the hierarchy. The issue is what 'pairing' is OK and what is not. For example
mammal-virus OK
mammal-bacteria OK
bacteria-bacteriophage OK
bacteriophage-mammal filter out

**Comment by sandraorchard** [comment]
I can write a fuller list tomorrow to start the discussion, if that helps.

**Comment by pporrasebi** [comment]
I think Pascale refers to the host information contained on the viral protein entry in UniProt. There we could potentially get a more accurate mapping of host-pathogen pairings. Whether it is worth integrating a parser for this kind of information in our export pipeline is another question. 

For my point of view, Sandra's approach would probably give more or less the same results, not sure if we need the specificity Pascale's approach brings in.

**Comment by sandraorchard** [comment]
Yes, sorry, didn't read it properly. I was thinking more generic filters would also allow us to get rid of all those saccharomyces-human interactions (we'd have to set that one lower than just fungi, as we'd otherwise loose genuine host-pathogen interactions) and assorted fly/worm-irrelevant species which I suspect are also in there. Should also be extended to the CC line export.

**Comment by pgaudet** [comment]
Hi @sandraorchard @pporrasebi 
The problem with the higher level mapping that Sandra suggests is that for example HIV proteins *could* interact with mouse proteins in vitro, but never in vivo since it's not a host. Depending on the types of experiments you capture they may be a high level of false positives with that approach. 

Pascale

**Comment by pgaudet** [comment]
Hi @sandraorchard @bmeldal @pporrasebi 
Any further thoughts on this ?

**Comment by pporrasebi** [comment]
We need the host-pathogen pairings from GO or UniProt. Once we get this, we can show you how to create the filters needed to select which interspecies interactions to discard and decide together where the filters need to be implemented.

**Comment by sandraorchard** [comment]
For viral proteins, there is a line (OH) in the flat file describing the host , when known.

OS   Sapporo virus (isolate GI/Human/Germany/pJG-Sap01)
OS   (Hu/Dresden/pJG-Sap01/DE).
OC   Viruses; ssRNA viruses; ssRNA positive-strand viruses, no DNA stage;
OC   Caliciviridae; Sapovirus.
OX   NCBI_TaxID=291175;
**OH   NCBI_TaxID=9606; Homo sapiens (Human).**

I don't think this exists for anything other than viruses but I can find out

**Comment by pgaudet** [comment]
Sounds good. I'll keep this open. Thanks ! Pascale

**Comment by jimhu-tamu** [comment]
Sounds like we might want to add this to the things the multi-organism group is discussing. What do you think @pgaudet?

**Comment by pgaudet** [comment]
Sure ! I've added this to next week's agenda.
